### PR TITLE
Fixes #10585: Migrate to custom logging now in Foreman core.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This is intended to convert over to custom logging in Foreman core as part of https://github.com/theforeman/foreman/pull/2376

**Comment by ehelms** [comment]
This is ready for review with the corresponding Foreman PR -- the tests pass if you run the branches together in Jenkins but won't pass here until the Foreman PR is merged.

File: lib/katello/configuration/loader.rb [hunk 1]
```diff
@@ -94,7 +78,7 @@ def load_yml_file(file_path)
       end
 
       def environment
-        Rails.env.to_sym rescue raise 'Rails.env is not accessible, try to use #early_config instead'
+        #Rails.env.to_sym rescue raise 'Rails.env is not accessible, try to use #early_config instead'
```
Comment (jlsherrill) [inline lib/katello/configuration/loader.rb]: intentional?

Comment (ehelms) [inline lib/katello/configuration/loader.rb]: Sorta, but I fixed it in a better way and reverted this change.

**Comment by ehelms** [comment]
Updated based on your comment (and set tire_rest to false by default again).

**Comment by jlsherrill** [comment]
I see this when running tests:

/home/vagrant/git/katello/app/lib/katello/util/thread_session.rb:67: warning: toplevel constant Logging referenced by Katello::Foreman::Logging
.Organization::organization validation

over and over and over.  I think you may want to change the reference on that line to ::Foreman::Logging

**Comment by jlsherrill** [comment]
ACK after that one change

**Comment by ehelms** [comment]
Merging based on green test run of this PR -- http://ci.theforeman.org/job/test_katello_pull_request/6452/

**Pull request merged**